Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml

```diff
@@ -0,0 +1,1094 @@
+      tags:
+      - compliance
+      scriptName: ToTable
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "36"
+    scriptarguments:
+      columns:
+        simple: Severity,Title,Description
+      data:
+        simple: ${PrismaCloudCompute.Images.ComplianceIssues}
+      extend-context:
+        simple: ComplianceTable=.
+      title:
+        simple: Prisma Cloud Compute  Image Compliance
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 2205
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    fieldMapping:
+    - incidentfield: Prisma Cloud Compute Markdown
+      output:
+        simple: ${ComplianceTable}
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 7dc085aa-78a5-4cda-85a4-34a52e8996a3
+    type: title
+    task:
+      id: 7dc085aa-78a5-4cda-85a4-34a52e8996a3
+      version: -1
+      name: Case Details
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 8d9783d2-8e97-43e3-8c4b-898be3c924fe
+    type: title
+    task:
+      id: 8d9783d2-8e97-43e3-8c4b-898be3c924fe
+      version: -1
+      name: Vulnerability
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "46"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 1565
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: 322282a6-f431-42cb-8870-815cb2a23ca0
+    type: title
+    task:
+      id: 322282a6-f431-42cb-8870-815cb2a23ca0
+      version: -1
+      name: Compliance
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 2060
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: b4fd8f72-715b-441a-84c9-dbd6efb3ad7f
+    type: regular
+    task:
+      id: b4fd8f72-715b-441a-84c9-dbd6efb3ad7f
+      version: -1
+      name: Get Defender Policy
+      description: DefenderSettings returns the defenders settings
```

Comment:
```suggestion
      description: DefenderSettings returns the defenders settings.
```
Maybe this description should be something like:
DefenderSettings returns the settings in the Defender policy.

Reply:
agreed!